Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM govukpay/openjdk:alpine-3.9-jre-base-8.201.08
+FROM adoptopenjdk/openjdk11:jdk-11.0.2.9-alpine
```

Comment:
We should ascertain what the differences are between the `govukpay` OpenJDK Alpine image and the standard OpenJDK Alpine image e.g. does it have extra packages installed? We probably want to replicate any differences with the AdoptOpenJDK Alpine image.

Reply:
👍 yes, sorry I should have added a note that I raised this PR to see if it would work and investigate any issues. They'll be more rigour around switching the base image before I actually do it.

Reply:
In that case, can we try using the slim images?